### PR TITLE
EF-152: Migrate signing to Garasign

### PR DESCRIPTION
**Pull request by adelinowona**

### CODE REVIEW

File: evergreen/evergreen.yml [hunk 1]
```diff
@@ -238,9 +238,8 @@ functions:
         include_expansions_in_env:
           - "ARTIFACTORY_PASSWORD"
           - "ARTIFACTORY_USERNAME"
-          - "AZURE_NUGET_SIGN_TENANT_ID"
-          - "AZURE_NUGET_SIGN_CLIENT_ID"
-          - "AZURE_NUGET_SIGN_CLIENT_SECRET"
+          - "GRS_USERNAME"
+          - "GRS_PASSWORD"
```
Comment (damieng) [inline evergreen/evergreen.yml:242]: Is `include_expansions_in_env` a shorter-hand syntax for the env format used in 1387? with `key: ${KEY}` ?

**Review by damieng — APPROVED** [review]
Okay to merge if answer to question is yes.

File: evergreen/evergreen.yml (hunk 1 above)
Comment (adelinowona) [reply evergreen/evergreen.yml:242]: yep.

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**